pkg/wallet/crypt.go

```diff
@@ -0,0 +1,68 @@
+package wallet
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"crypto/sha256"
+	"fmt"
+	"io"
+)
+
+type crypt struct {
+	key []byte
+}
+
+func NewCrypt(key []byte) *crypt {
```

Comment:
I suggest to use https://godoc.org/golang.org/x/crypto/argon2 for password hashing instead of hash5000